### PR TITLE
Events optimizations

### PR DESCRIPTION
**Pull request by yasiupl**
via https://forum.hsp.sh/t/hackerspejsowy-nowy-lad/240/8

### CODE REVIEW

**Review by not7cd — APPROVED** [review]

File: netlify.toml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+[[redirects]]   
+  from = "/calendar.ics"
+  to = "https://wydarzenia.hsp.sh/@hs/feed/ics"
+  
+ [[redirects]]
```
Comment (not7cd) [inline netlify.toml:5]: leading space

**Review by not7cd — CHANGES_REQUESTED** [review]

**Review by not7cd — COMMENTED** [review]
/calendar is not added to any menu

File: calendar.md [hunk 2]
```diff
@@ -3,9 +3,17 @@ layout: page
...
\ No newline at end of file
+<iframe src="https://gra0007.github.io/modern-cal-embed/iframe.html?ical=https%3A%2F%2Fwydarzenia.hsp.sh%2F%40hs%2Ffeed%2Fics&title=1&nav=1&date=1&view=1&details=0&monstart=1&dview=0&color=%2300cc99&colorbg=%231b1b1b&colortxt=%23ffffff&colorsecondarytxt=%23FFFFFF"
+			width="100%" height="400px" style="border: 1px solid #ffffff"></iframe>
+
+
+Link do kalendarza w formacie: [ICS/WebCal](/calendar.ics), [RSS/Atom](/calendar.atom). Możesz dodać je do swojego kalendarza lub klienta RSS!
+
+`https://hsp.sh/calendar.ics`
+
+`https://hsp.sh/calendar.atom`
```
Comment (not7cd) [inline calendar.md:19]: unnecessary

**Review by not7cd — CHANGES_REQUESTED** [review]

File: calendar.md (hunk 2 above)
Comment (yasiupl) [reply calendar.md:19]: Not everyone might realize that you can right click copy the URL. Mind you, that the URLs don't open in new tab, rather download as a file.

Comment (not7cd) [reply calendar.md:19]: Lets instruct user to copy the URL and teach them something new about the web. We could link them to a guide (external, or wiki?).

Comment (yasiupl) [reply calendar.md:19]: That's more complicated than simply putting the URL in plain text to view. The wiki instruction can always be added later.

Comment (not7cd) [reply calendar.md:19]: Then maybe remove `<a>` links and place two read only `<input type="text" readonly>` with URLs as values. That would communicate purpose clearly and simplify selecting and copying.

**Pull request merged**